spyderlib/qt/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 import os
 
-os.environ.setdefault('QT_API', 'pyqt')
+os.environ.setdefault('QT_API', 'pyqt5')
```

Comment:
@flying-sheep, giving your opposition to revert this change, I'm going to revert it myself after merging.

It's not up to you to decide when we need to move to fully support to PyQt5, it's up to us :-) And right now, it's a consensus among the dev team that we are just not ready.